packages/content/test/attributes/IdentityAttribute.test.ts

```diff
@@ -259,4 +259,17 @@ describe("IdentityAttribute", function () {
         expect(affiliationAttribute.value.role).toBeInstanceOf(AffiliationRole);
         expect(affiliationAttribute.value.unit).toBeInstanceOf(AffiliationUnit);
     });
+
+    test("should validate tags", function () {
```

Comment:
Since the validation method for the `tags` validates more than just uniqueness, you could choose a more specific test name.

```suggestion
    test("should validate uniqueness of tags", function () {
```